lib/Test2/Event/ParseError.pm

```diff
@@ -0,0 +1,20 @@
+package Test2::Event::ParseError;
+use strict;
+use warnings;
+
+our $VERSION = '0.000012';
+
+BEGIN { require Test2::Event; our @ISA = qw(Test2::Event) }
```

Comment:
Given the drama surrounding base.pm and parent.pm, I'd say if you don't *need* to use either of them, and can avoid their use easily and elegantly, do so.

Using them simply makes the total complexity go up slightly without changing the effective behaviour.